wordpress-mobile/WordPress-Android
# Remove WP Stories feature config and the unused code

Fixes #14318 

This PR removes the WPStoriesEnabled flag because it was already released to all the users and it will not be reverted. The code should be quite straightforward.

To test:
- Check you can correctly add stories on a WPCom site
- Check the stories are disabled on a non-Jetpack self-hosted site

PR submission checklist:
